src/frontend/js/utils/api/web-analytics/google_analytics.spec.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { ContextFactory as mockContextFactory } from 'utils/test/factories';
+import WebAnalyticsAPIHandler from '.';
+import GoogleAnalyticsApi from './google_analytics';
+
+jest.mock('utils/context', () => ({
+  __esModule: true,
+  default: mockContextFactory({
+    web_analytics_provider: 'google_analytics',
+  }).generate(),
+}));
+
+// Mock the `ga` function so the verification of the presence of the Google Analytics passes.
+// eslint-disable-next-line func-names
+global.ga = function (
+  tracker: string,
+  hitType: string,
+  eventCategory: string,
+  eventAction: string,
+  eventLabel: string,
+  eventValue?: any,
+) {};
```

Comment:
You should move this declaration into a `beforeAll` within the `describe` block.

Reply:
Done